apps/blockchain/test/blockchain/state_test.exs

```diff
@@ -471,7 +471,7 @@ defmodule Blockchain.StateTest do
 
   test "Blockchain state tests" do
     Enum.each(test_directories(), fn directory_path ->
-      test_group = Enum.fetch!(String.split(directory_path, "/"), 4)
+      test_group = Enum.fetch!(String.split(directory_path, "/"), -1)
```

Comment:
can we use pipe  operator?

Reply:
Yeah, we definitely can. I'll update this.

Reply:
Updated in https://github.com/poanetwork/mana/pull/367/commits/ab7ed76b0478dda896fb450f5b7449178ad167f1